android/src/main/java/com/swmansion/gesturehandler/core/ManualGestureHandler.kt

```diff
@@ -3,6 +3,8 @@ package com.swmansion.gesturehandler.core
 import android.view.MotionEvent
 
 class ManualGestureHandler : GestureHandler<ManualGestureHandler>() {
+  override fun isContinuous() = false
```

Comment:
I'm not sure if we should treat `Manual` as `discrete`. This gesture is very specific and could act as both

Reply:
I was planning to add a JS-side property to set if a given `Manual` should be treated as `discrete` or `continuous`.
But I agree that by default it makes more sense for it to be `continuous`, will fix this.